<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Sonie" failures="1" tests="5" name="Test" time="198.395" errors="0" timestamp="2022-02-26T03:15:58 IST">
  <testcase classname="tests.FourTest" name="testFour" time="24.297">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.FourTest.testFour(FourTest.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- testFour -->
  <testcase classname="tests.TwoTest" name="testTwo" time="20.945"/>
  <testcase classname="tests.LoginTest" name="login" time="11.031"/>
  <testcase classname="tests.LoginTest" name="login" time="14.325"/>
  <testcase classname="tests.ThreeTest" name="testThree" time="56.893"/>
</testsuite> <!-- Test -->
